static/app/views/settings/projectDebugFiles/externalSources/customRepositories/details.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from '@emotion/styled';
+
+// import DateTime from 'app/components/dateTime';
+import NotAvailable from 'app/components/notAvailable';
+import {t, tct} from 'app/locale';
+import space from 'app/styles/space';
+import {AppStoreConnectValidationData} from 'app/types/debugFiles';
+
+type Props = {
+  details?: AppStoreConnectValidationData;
+};
+
+function Details({details}: Props) {
+  const {latestBuildVersion, latestBuildNumber} = details ?? {};
+  return (
+    <Wrapper>
+      {t('Last detected version')}
+      <Value>
+        {latestBuildVersion ? (
+          tct('v[version]', {version: latestBuildVersion})
+        ) : (
+          <NotAvailable tooltip={t('Not available')} />
+        )}
+      </Value>
+
+      {t('Last detected build')}
+      <Value>{latestBuildNumber ?? <NotAvailable tooltip={t('Not available')} />}</Value>
+
+      {/* {t('Detected last build on')}
```

Comment:
The comments will be removed as soon as the backend provides the correct data format